### ISSUE TITLE
video-swap-new (userscript) woes

### ISSUE THREAD

**Issue by MonfGeiger** [pixeltris/TwitchAdSolutions#263]
Firefox: 127.0 (64-Bit, Stable channel)
Block Method: video-swap-new (userscript, 1.26)
Using Tampermonkey

TL;DR i took the most recent update to this userscript but stuck with TTV LOL PRO (v2) because idk if the userscript update worked. today, i turned the extension off and the userscript back on, and let it run for a hot minute or two, and it worked fine.....then this happened

![Capture](https://github.com/pixeltris/TwitchAdSolutions/assets/28603306/ba7d5fe1-b2e8-441c-a245-9637706ebf3c)

guess the error #1000 upgraded to a #2000

I did clear my browser cache, even used ccleaner to selectively delete ALL twitch cookies, then tried again, no dice, more error #2000

Edit: tried updating my userscripts, saw that video-swap-new updated to 1.27, will try that out and respond soon

**Comment by MonfGeiger** [comment]
never mind, there was another update that brought the userscript to 1.27, and there's no more error 1000 or 2000 warnings.
